.github/setup-node/action.yml

```
@@ -0,0 +1,50 @@
+name: 'Setup Node'
+description: 'Setup node and install dependencies with caching support'
```

```suggestion
description: 'Setup Node.js and install npm dependencies with caching support'
```

Can we change all occurrences of "NodeJS", "node" etc to "Node.js"? Also, NPM is an acronym, but it's actually `npm` (all [lower case](https://www.npmjs.com/about)). I [recently updated this in WP Core to be consistent](https://core.trac.wordpress.org/ticket/56816).